scripts/error_codes.py

```diff
@@ -220,15 +220,15 @@ def examine_id_coverage(top_dir, source_id_to_file_names, new_ids_only=False):
             return False
 
     old_source_only_ids = {
-        "1123", "1218", "1220", "1584", "1823", "1950",
-        "1988", "2461", "2512", "2592", "2657", "2800", "2842", "2856",
-        "3046", "3263", "3356", "3441", "3682", "3876",
-        "3893", "3996", "4010", "4281", "4802",
-        "5052", "5073", "5170", "5188", "5272", "5347", "5473",
-        "5622", "6041", "6084", "6272", "7110", "7128", "7186",
-        "7589", "7593", "7653", "7812", "7885", "8065", "8084", "8140",
-        "8261", "8312", "8592", "8758", "9011",
-        "9085", "9390", "9440", "9547", "9551", "9615", "9980",
```

Comment:
Many of the ids removed here do not have new tests. Am I missing anything?

Reply:
@chriseth @axic what about the removed ids?

Reply:
They were already tested somewhere, just not removed from the list.

Reply:
Ah ok